### PR TITLE
Mobile version project

### PR DESCRIPTION
**Pull request by Fayob**
This is a portfolio setup project which is the mobile aspect of the project, the following includes what I did in the project.

I added html file (the markup document), these entails all the structure for the websites
I added style file, which entails the styles for the whole project
I setup the linters to check if some specific features were added to the website for better user experience
I added .gitignore file to ignore some file and folder
I added README file which entails the description about the project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+        </div>
+        </article>
+        </div>
+    </section>
+    <!-- The About Article -->
+    <article class="about_me">
+    <div class="about_me_background">
+            <h2>About me</h2>
+      <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you
+        see and have a project you need coded,
```
Comment (Ugboaja-Uchechi) [inline index.html]: - [ ] Great job using semantic HTML code 👏 . Please wrap your `article` tag in a `section` tag as the about me is a new section of your webpage 🙏 .

File: style.css [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+}
+
+.card {
+  height: 474px;
+  width: 327px;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+  margin: 0.5rem 0;
+  margin-left: -22px;
+}
```
Comment (Ugboaja-Uchechi) [inline style.css]: - [ ] Please make sure your project cards are centered as seen in the Figma design 🙏 
- [ ] Kindly give your project cards a border as seen in the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
image  |  image

File: style.css [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+  margin: 1rem;
+  padding: 1rem;
+  width: 327px;
+}
+
+.header_container {
+  padding: 0.5rem;
+  font-family: sans-serif;
+  display: flex;
+  align-items: center;
```
Comment (Ugboaja-Uchechi) [inline style.css]: - [ ] Please make sure you use the right `font family` specified in your chosen Figma design 🙏

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @Fayob, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (Fayob) [reply style.css]: The font family specified in the figma design is not in css font family

Comment (Ugboaja-Uchechi) [reply style.css]: Hi @Fayob , you can search for the font required in [Google font](https://fonts.google.com/). After you have found the font family, choose the font weight required. Then import it and add the font family to your code

Comment (Fayob) [reply style.css]: okay, Thanks

File: style.css [hunk 4]
```diff
@@ -9,11 +9,11 @@ body {
   margin: 1rem;
   padding: 1rem;
   width: 327px;
+  font-family: 'Inter', sans-serif;
 }
```
Comment (mukhammadrizooff) [inline style.css]: - [ ] [OPTIONAL] It looks like there is a problem with your project that needs a small fix. You can see in the picture that there is more space on the right side of the project than on the left side, which is not good. please try to fix this!

image

**Review by mukhammadrizooff — APPROVED** [review]
Hi @Fayob, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 
- Good communicate with reviewer ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**